scripts/dune_wrapper

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
it would have been nice to see dune_wrapper as an actual tool written in OCaml that we can maintain and improve :o 

at the very least it'd be nice to move it outside of `scripts/` as the directory seems to have become a landfill for bash scripts